airbyte-config/init/src/main/resources/seed/source_specs.yaml

```diff
@@ -6607,39 +6607,3 @@
     supportsNormalization: false
     supportsDBT: false
     supported_destination_sync_modes: []
-- dockerImage: "airbyte/source-zenloop:0.1.0"
```

Comment:
seems like this entry should not be removed

Reply:
This file is generated automatically, when you building the PLATFORM, or using this command: 
`./gradlew :airbyte-config:init:processResources`. Therefore I didn't change that. Manual redo of the mentioned command didn't add any of additional entities into `source_spec.yaml`.